src/OnlineSales/DTOs/FileCreateDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿// <copyright file="FileCreateDto.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+
+namespace OnlineSales.DTOs;
+
+public class FileCreateDto
+{
+    [Required]
+    public IFormFile? File { get; set; }
```

Comment:
It must be possible to customise max possible files size similar to how it is done in MediaExtensionAttribute.

Reply:
FileExtensionAttribute added